src/main/scala/edu/berkeley/ce/rockslicing/BlockVTK.scala

```diff
@@ -0,0 +1,197 @@
+package edu.berkeley.ce.rockslicing
+
+import breeze.linalg
+import breeze.linalg.{DenseVector, DenseMatrix}
+
+object BlockVTK {
+
+  private val EPSILON = 1.0e-6
```

Comment:
Yeah, I figured that would be the case. Should be a pretty quick fix. Thought about rebasing the `Numerics` branch, but it's such a small amount of code to change that we can just do it manually after.